db/migrate/20210624134755_add_slug_to_cips.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+SLUGS = {
+  "Ambition Institute": "ambition",
+  "Education Development Trust": "edt",
+}.freeze
```

Comment:
Probably need more

Reply:
UCL -> ucl
Teach First -> teach-first

are both basically what we want already so no need to override them